class InsertError(Exception):
    pass

class NotFoundError(Exception):
    pass
